### PR TITLE
feat: css/parts

### PR DESCRIPTION
**Pull request by git-anurag-hub**
## Description

Fixes #(issue)

## Checklist

- [ ] My code follows the [contributing guidelines][contributing] of this project.
- [ ] I am aware that all my commits will be squashed into a single commit, using the PR title as the commit message.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have updated the user-facing documentation to describe any new or changed behavior.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have not reduced the existing code coverage.

## Comments

[contributing]: CONTRIBUTING.md

### CODE REVIEW

File: packages/ecc-utils-design/src/components/form/form.ts [hunk 1]
```diff
@@ -173,165 +146,157 @@ export class Form extends LitElement {
...
+        )}
+      </div>
+    `;
+  }
+
+  renderTemplate(field: Field, path: string): TemplateResult {
+    if (field.type === "array") {
+      return this.renderArrayTemplate(field, `${path}.${field.key}`);
     }
     if (field.type === "switch") {
```
Comment (JaeAeich) [inline packages/ecc-utils-design/src/components/form/form.ts:287]: else if?

**Review by JaeAeich — APPROVED** [review]

**Comment by JaeAeich** [comment]
@git-anurag-hub if possible I think we should also add `object` as a type to Field, this way we can reduce the data parsing at the client side.

```
```

**Comment by JaeAeich** [comment]
It doesn't make sense to have labels in an array, maybe we should have `parentLabel + index` as `label` and index as key in such cases.  On clicking plus will just renders another input fields in such cases. What do you think?

**Comment by git-anurag-hub** [comment]
> It doesn't make sense to have labels in an array, maybe we should have `parentLabel + index` as `label` and index as key in such cases. On clicking plus will just renders another input fields in such cases. What do you think?

Yeah that do make sense! But lets focus on moving fast I guess this will not be a big problem for the version 1, we will definitely come to this in future.

**Comment by git-anurag-hub** [comment]
Hey @SalihuDickson & @JaeAeich, I hope you have read the docs here. If not you can see them in the latest deployment of this PR. I will write more docs soon!

**Pull request merged**